pkg/cmd/roachtest/tests/mixed_version_backup.go

```diff
@@ -1473,14 +1477,22 @@ func (u *CommonTestUtils) setClusterSettings(
 		return nil
 	}
 
+	// Make sure to use a connection to the system tenant as most of
```

Comment:
> most

So some of the settings are tenant accessible, but we use system anyway? For simplicity reasons (which seems reasonable to me) or something else?

Reply:
Heh, you're right, it's more like like 50-50. I pushed something to actually differentiate between the two of them, I'll run this this a few times before merging.